.github/workflows/preview_link.yml

```diff
@@ -29,12 +29,15 @@ jobs:
     - name: Generate links
       env:
         branch: ${{ github.head_ref }}
+        RENAMED_FILES: ${{ steps.changed_files.outputs.renamed_files }}
+        MODIFIED_FILES: ${{ steps.changed_files.outputs.modified_files }}
+        ADDED_FILES: ${{ steps.changed_files.outputs.added_files }}
       id: comment_body
       run: |
         python local/bin/py/preview_links.py --deleted='${{ steps.changed_files.outputs.deleted_files }}' \
```

Comment:
We don't need it for deleted files here?

Reply:
Good catch

Reply:
Done in 13a8b35